### ISSUE TITLE
[BUG] LITE Network : parametrize with list arguments

### ISSUE THREAD

**Issue by Cyril-Meyer** [aeon-toolkit/aeon#1867]
### Describe the feature or idea you want to propose

PR #1851 introduce new tests for networks. This made it possible to highlight a network not functioning as expected : [LITE](https://github.com/aeon-toolkit/aeon/blob/6bd3324624dda8835fa17579e6575c470d287c5a/aeon/networks/_lite.py). We can also note that the [TapNet](https://github.com/aeon-toolkit/aeon/blob/6bd3324624dda8835fa17579e6575c470d287c5a/aeon/networks/_tapnet.py) does not behave like the other networks with kernel_size and padding parameters.

For now, those networks are ignored by the tests [code ref](https://github.com/aeon-toolkit/aeon/blob/6bd3324624dda8835fa17579e6575c470d287c5a/aeon/networks/tests/test_all_networks.py#L108-L116).
```python
        # Exceptions to fix
        if (
            attrname in ["kernel_size", "padding"]
            and network.__name__ == "TapNetNetwork"
        ):
            continue
        # LITENetwork does not seem to work with list args
        if network.__name__ == "LITENetwork":
            continue
```

The doc already states that list can be used, e.g. : 
```
n_filters : int or list of int32, default = 32
        The number of filters used in one lite layer, if not a list, the same
        number of filters is used in all lite layers.
```
List parameterization is just not yet implemented like on other networks.

### Describe your proposed solution

Implement list parametrization for LITE as for other networks e.g. Inception : 
```python
        if isinstance(self.n_filters, list):
            assert len(self.n_filters) == self.depth
            self._nb_filters = self.n_filters
        else:
            self._nb_filters = [self.n_filters] * self.depth

[.....]

        for d in range(self.depth):
[.....]
            x = self._inception_module(
                x,
                n_filters=self._nb_filters[d],
[.....]
```

### Describe alternatives you've considered, if relevant

Another possibility, is to update the doc and to don't allow the usage of lists for the arguments.
In my opinion, this is a bad idea.

**Comment by dschrempf** [comment]
The same is true for `kernel_size`. The documentation states

> kernel_sizeint or list of int, default = 40
The head kernel size used for each lite layer, if not a list, the same is used in all lite module.

But using a list leads to an error
```
TypeError: unsupported operand type(s) for //: 'list' and 'int'
```

**Comment by MatthewMiddlehurst** [comment]
Seems like either a bug or documentation issue, depending on how its resolved 🙂. @hadifawaz1999 is the author but is a bit busy currently, good to hear thoughts though.

**Comment by hadifawaz1999** [comment]
Its a doc issue, should just not mention lists when it comes to LITE

**Comment by kavya-r30** [comment]
@MatthewMiddlehurst  @hadifawaz1999 should I approach this by removing list from the docs in _lite.

**Comment by hadifawaz1999** [comment]
> [@MatthewMiddlehurst](https://github.com/MatthewMiddlehurst) [@hadifawaz1999](https://github.com/hadifawaz1999) should I approach this by removing list from the docs in _lite.

yes from _lite in network from _lite_time from classification and regression deep learning
